Dockerfile.prod

```diff
@@ -0,0 +1,38 @@
+FROM node:latest as webpack
```

Comment:
We should perhaps not run latest node on prod? I think a LTS-version would be more suitable, e.g. `node:12`, would probably be fine either way, just a lil bit worried in terms of compability with our somewhat outdated frontend-dependencies.

Otherwise great stuff! 🍨 

Also: there are several files here that are lacking newlines at the end, and I think we should add it _just_ to be sure, you know?

Reply:
The stage using node is just a build stage, but yes, I agreee. I have updated the PR now.